examples/hello-assembly/test.sh

```diff
@@ -0,0 +1,9 @@
+asc -v || npm install --save-dev AssemblyScript/assemblyscript
+
+asc hello-assembly.ts -b test.wasm -t test.wast
+
+node -i -e "log_char = c => process.stdout.write(String.fromCodePoint(c));\
+  new WebAssembly.Instance(new WebAssembly.Module(require('fs').readFileSync('test.wasm')),{console:{log_char}})"
```

Comment:
While I understand this, it doesn't look simple. Could you please split it in multiple lines with a few comments?